skimage/io/collection.py

```diff
@@ -416,39 +416,36 @@ class MultiImage(ImageCollection):
 
     Notes
     -----
-    The object that is returned can be used as a list of image-data objects,
-    where each entry in the list represents one image. In this regard it is
-    very similar to `ImageCollection`, but the two differ in the treatment
-    of multi-frame images.
+    The object that is returned is a list of image-data arrays. In this
+    regard, it is very similar to `ImageCollection`, but the two differ in
+    their treatment of multi-frame images.
 
     For a TIFF image containing N frames of size WxH, `MultiImage` stores
-    all frames of that image as a single entry of shape `(N, W, H)` in the
-    list. `ImageCollection` instead creates N entries of shape `(W, H)`.
+    all frames of that image as a single element of shape `(N, W, H)` in the
+    list. `ImageCollection` instead creates N elements of shape `(W, H)`.
 
-    For an animated GIF image, `MultiImage` in the current implementation
-    will only read one frame, while `ImageCollection` by default will read
-    all of them.
+    For an animated GIF image, `MultiImage` reads only the first frame, while
```

Comment:
That's what I noticed with `no_time_for_that_tiny.gif` -- hoping it's not any random one that `MultiImage` loads...

Reply:
@mkcor for the GIFs that I have tried, `MultiImage` indeed seems to load the first frame. At the moment I am not even sure `ImageCollection` does the right thing for animated GIF. Could it be that it detects all the frames but always returns the first one? I will try to look into this in the context of [#6291](https://github.com/scikit-image/scikit-image/pull/6291) once I have time.